src/components/Notifications/form/event.ts

```diff
@@ -396,6 +396,25 @@ export const getEventSchema = (
               ),
             },
           },
+          {
+            properties: {
+              eventSelect: {
+                enum: [ProjectTriggers.SUCCESSFUL_TASK_EXCEEDS_DURATION],
+              },
+              extraFields: {
+                type: "object" as "object",
+                title: "",
+                required: [ExtraFieldKey.TASK_DURATION_SECS],
+                properties: {
+                  [ExtraFieldKey.TASK_DURATION_SECS]: taskDurationInput,
+                },
+              },
+              regexSelector: regexSelector(
+                regexEnumsToDisable,
+                taskRegexSelectors
+              ),
+            },
+          },
```

Comment:
Since it seems like this trigger shares the same fields as                   `ProjectTriggers.TASK_EXCEEDS_DURATION`, you can actually just delete this block and combine it with the above block so that it looks like:
```
       {
            properties: {
              eventSelect: {
                enum: [
                  ProjectTriggers.TASK_EXCEEDS_DURATION,
                  ProjectTriggers.SUCCESSFUL_TASK_EXCEEDS_DURATION,
                ],
              },
              ... blah blah blah
          },
```